node/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:alpine
+
+WORKDIR /opt/app
+COPY . /opt/app
+RUN npm install
+
```

Comment:
I think adding 

```
COPY --chown=node:node . .
```

Is good practice too for node, per: https://www.digitalocean.com/community/tutorials/how-to-build-a-node-js-application-with-docker#step-3-%E2%80%94-writing-the-dockerfile

Reply:
:zap: